src/parser/mod.rs

```diff
@@ -11411,6 +11411,9 @@ impl<'a> Parser<'a> {
             Token::Colon if self.dialect.supports_named_fn_args_with_colon_operator() => {
                 Ok(FunctionArgOperator::Colon)
             }
+            Token::Word(w) if w.value.eq_ignore_ascii_case("value") => {
```

Comment:
would it make sense to have value be a keyword so that this can potentially become `if self.parse_keyword(Keyword::VALUE)`?

Reply:
Yes, I guess so, but I think this is out of the scope of this PR...

Reply:
hmm not sure I follow by it being out of scope. Took a quick look and VALUE is also already a keyword so that the PR doesn't need to touch that file either. thinking the changes required in the pr could be something like this to use the convention vs comparing the raw string?
```rust
if self.parse_keyword(Keyword::VALUE) {
    return Ok(FunctionArgOperator::Value)
}
```